.github/workflows/pr-env.yml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'External') == false
```

Comment:
I was a couple seconds late.
Wouldn't it make more sense to expand this to all external PR, regardless of the label?
@oreonl @iethree 

`github.event.pull_request.head.repo.full_name == github.repository`